packages/adapter-vercel/index.d.ts

```diff
@@ -4,6 +4,36 @@ type Options = {
 	edge?: boolean;
 	external?: string[];
 	split?: boolean;
-};
+} & Config;
 
 export default function plugin(options?: Options): Adapter;
+
+export interface Config {
+	/**
+	 * To which runtime to deploy the app. Can be one of:
+	 * - `edge`: https://vercel.com/docs/concepts/functions/edge-functions
+	 * - `serverless` or a string specifying the serverless runtime (like `node16`): https://vercel.com/docs/concepts/functions/serverless-functions
+	 * @default 'serverless'
+	 */
+	runtime?: 'serverless' | 'edge' | (string & {});
```

Comment:
I think this should probably match the build output API

```suggestion
	/**
	 * Whether to use [Edge Functions](https://vercel.com/docs/concepts/functions/edge-functions) or [Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions)
	 * @default 'nodejs18.x'
	 */
	runtime?: 'edge' | 'nodejs16.x' | 'nodejs18.x';
```